web/src/components/nomination/NominateForm.vue

```diff
@@ -48,40 +83,96 @@ export default {
     NameInput,
     JobInput,
     LocationInput,
+    UrlsInput,
     SubmitterInput,
     TopicsInput,
   },
   data() {
     return {
+      languageOptions: ['English', '日本語'],
       form: {
-        speakerName: {
-          title: '',
-          first: '',
-          middle: '',
-          last: '',
+        name: {
+          en: '',
+          ja: '',
         },
-        speakerEmail: '',
-        firstJob: {
+        email: '',
+        photo_url: '',
+        job: {
           title: '',
           company: '',
         },
-        speakerBio: '',
+        speaker_bio: '',
         location: {
           city: '',
           prefecture: '',
         },
-        submitterInput: {
+        urls: {
+          linkedin: '',
+          twitter: '',
+          facebook: '',
+          website: '',
+        },
+        submitter: {
           name: '',
           email: '',
         },
         topics: [],
-        permisssion: false,
+        languages: [],
+        consent: false,
       },
     };
   },
   methods: {
-    submit() {
-      // TODO
+    handleSubmit() {
+      const payload = this.parseFormData();
+      if (process.env.NODE_ENV === 'production') {
+        this.$db('People').create(payload, this.afterSave);
+      } else {
+        console.log(payload);
+      }
+    },
+    // parse the data into the payload format expected by Airtable
+    parseFormData() {
+      /* eslint-disable camelcase */
+
+      // copy the the fields that don't need parsing from the form object
+      const {
```

Comment:
🤔 We have three different ways of moving items from our form Object into our payload object. It might be more readable to simply assign each one like so: 

```
payloadFields = {
       email := this.form.email,
        job_title: this.form.job.title,
        company: this.form.job.company,
        city: this.form.location.city,
        location_id: [this.form.location.prefecture],
      };
```

Especially the Object.keys part is confusing because sometimes we're looking at things with a prefix, and other times we're looking for things with a suffix. It's harder to read the code and ensure we're setting all the right values.

Reply:
Yeah, this parsing is a bit of an over-complication.... I was wondering, do you think it actually makes sense to just change the `form` object to match the payload final format and then just assign directly there when the inputs are changed? This way we can get rid of parsing altogether, I'm thinking it makes more sense and make it simpler? What do you think?

Reply:
I agree with the caveat that merging my validation PR will be a lot more complex. Can we build the payloadFileds and then as a future refactor, restructure the form?